message_buffer/programs/message_buffer/src/instructions/resize_buffer.rs

```diff
@@ -31,13 +31,11 @@ pub fn resize_buffer<'info>(
         .is_allowed_program_auth(&allowed_program_auth)?;
```

Comment:
Unrelated to this PR: but now that the message buffer account is only 1, should we move it to the Anchor context?

Reply:
@guibescos i was wrong about this. I was confusing the issues from working with an older implementation. I made a new PR to address this